<launch>
    <!-- Params -->
    <arg name="max_thrust_limit" default="100"/>
    <arg name="min_thrust_limit" default="50"/>


    <!-- DESIRED_THRUST/RUDDER to THRUST_L/R -->
    <node pkg="thrust_control" type="thrust_control.py" name="thrust_control_node" output="screen">
        <param name="max_thrust_limit" type="double" value="$(arg max_thrust_limit)"/>
        <param name="min_thrust_limit" type="double" value="$(arg min_thrust_limit)"/>
    </node>

</launch>
